src/ms.tex

```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
 
 \secauthor{Aarya Patil}
 
-Summarize funding sources (Moore, NASA) and amounts and what this has been used
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
+of the project. The Astropy project accepts funds from institutions as well as
+individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that
+supports and promotes world-class, innovative, open source scientific computing.},
+the fiscal sponsor of the project. Previously, no direct financial support
+was available for the project development, and NUMFOCUS covered the incurred
```

Comment:
NumFOCUS didn't cover all (or any?) incurred costs.

Reply:
I took this from the Astropy v2.0 paper: https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f/pdf -- 2.6. Support of Astropy "NumFOCUS has, to date, covered the direct costs incurred by the Astropy Project." Is that not correct? Should I use the word "most" or "some"?